process/smartContract/hooks/blockChainHook.go

```diff
@@ -630,6 +658,12 @@ func (bh *BlockChainHookImpl) RevertToSnapshot(snapshot int) error {
 	return bh.accounts.RevertToSnapshot(snapshot)
 }
 
+// EpochConfirmed is called whenever a new epoch is confirmed
+func (bh *BlockChainHookImpl) EpochConfirmed(epoch uint32, _ uint64) {
+	bh.flagSaveAccountsIfError.Toggle(epoch < bh.saveAccountsIfErrorDisableEpoch)
+	log.Debug("blockchain hook: save accounts if error", "enabled", bh.flagSaveAccountsIfError.IsSet())
```

Comment:
log.Debug("blockchain hook: built in functions save accounts if error"...

Reply:
done